### ISSUE TITLE
Redirect to Plaid?

### ISSUE THREAD

**Issue by sarink** [plaid/link#240]
Does Plaid offer a system where my users can _at least_ redirect to Plaid.com and enter their bank credentials there (in an oauth-like fashion)?

How is any user supposed to trust this startup company (Plaid's business model was, "go create fintech startup apps!", right?) that they've never heard of and provide their bank details in a modal on weirdnewcompany.com? This seems like an extremely crucial flaw...

(I don't mean to sound so judgey, I think Plaid's great and you guys have done a lot of awesome work, I do think this is a giant problem which could be fixed relatively easily though, so I'm curious why it doesn't exist)

**Comment by pbernasconi** [comment]
Hey @sarink – We completely understand your reasoning and questions around having Plaid Link sit inside of our customers applications, as opposed to a more OAuth-like flow. OAuth is becoming a more standard approach to 3rd party integrations and services, and it was a path we considered when we first built Link.

Plaid Link however is more dynamic and suitable for mobile traffic however, where our customers can embed it inside of their applications, and control the entire experience when their users link their bank accounts to their apps. OAuth can be a rather jarring experience, and would disrupt the onboarding flow of many of our customers applications, whereas Plaid Link fits nicely at any point in your app and is now customizable via our Dashboard.

To answer your question of "How is any user supposed to trust a company they've never heard of and provide their bank details in a modal on weirdnewcompany.com" – we've seen a lot of success when our customers prepare their end-users with some messaging and content that explains how Plaid works and what it means to connect their bank accounts to their app, before Link fully opens and users can link their bank. We're also hard at work at building trust with our end-users, and I would expect some more progress to made here in the near future.

**Comment by sarink** [comment]
@pbernasconi thanks for your quick response. I do agree that Link is a nicer experience for users, and if you're, say, Venmo, that's great. Users already trust you. But if you're a new company (part of Plaid's goal and mission statement is to help these new startup companies), there's absolutely no indication to a user that this is legitimate. Let's go as far to assume the user is familiar with Plaid and trusts Plaid, there's still no indication this isn't a phishing scam and that the application hasn't created a Plaid-Link-lookalike.

Personally, when I've encountered Plaid on other websites, I open the network tab and ensure that the modal was actually sourced from `cdn.plaid.com`. But obviously we cannot expect regular users to do this...

Even if a blurb was included to explain who Plaid is, and even if the user trusts Plaid, that isn't enough. 

I worked hard to build https://basilfinance.com, only to have many many users be scared off by this, and there's nothing I can do about it. It's a bummer.

I hope this feature request makes it into your queue someday :)

Thanks for building Plaid!
